includes/subscriptions.php

```diff
@@ -234,7 +234,25 @@ function delete_subscriptions( $post_id ) {
  * @since  1.0
  */
 function send_notifications( $post_id ) {
-	if ( ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) || wp_is_post_revision( $post_id ) || ! current_user_can( 'edit_post', $post_id ) ) {
+	if ( ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) || wp_is_post_revision( $post_id ) ) {
+		return;
+	}
+
+	if ( ! wp_doing_cron() ) { // @codingStandardsIgnoreLine `wp_doing_cron(..)` is a WP function
+		/**
+		 * Add possibility to send notification in background
+		 *
+		 * @param bool  false   Whether run 'send notification' in background or not, default 'false'
+		 * @param array $params request data
+		 */
+		$send_notification_in_background = apply_filters( 'dt_send_notification_allow_in_background', false, $post_id );
```

Comment:
this will affect all subscription updates for updated posts, these 'notifications' are required for updating pulled posts. This filter seems unrelated to the overall purpose of the PR, why is it included here?

Reply:
I'll take a look at this.

Reply:
@adamsilverstein basically, we are trying to prevent send any REST API request during post update and move them into background. What if we will just add a filter, somewhere on the top and prevent send notifications, maybe near 
```
if ( ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) || wp_is_post_revision( $post_id ) ) {
		return;
	}
```